source/options.test.js

```diff
@@ -0,0 +1,39 @@
+const $ = require('jquery');
+const html = `
+  <input id="claire_guide" type="checkbox"></input>
+  <input id="debug_log_checkbox" type="checkbox"></input>
+`;
+
+test('update localStorage', () => {
+  global.localStorage = {};
+  document.body.innerHTML = html;
+  const options = require('./options');
+
+  options.run();
+
+  expect($('#claire_guide').click().prop('checked')).toBe(true);
+  expect(localStorage.hide_guide).toBe('yes');
+
+  expect($('#debug_log_checkbox').click().prop('checked')).toBe(true);
+  expect(localStorage.debug_logging).toBe('yes');
+
+  expect($('#claire_guide').click().prop('checked')).toBe(false);
```

Comment:
This test and the test below it implicitly depend on the state set by the tests above, would be good to leave a comment about that part.

In a future iteration, think it'd be better to have these be laid out as a sequence of events that are expected to change state starting with the initial state expectations - similar to how you do this in the _renders from localStorage_ test below.

Thinking of something like this:

Do an initial state check based on the initialization 
<element>.prop('checked')).toBe(false);
<trigger click event>
check for expected state on element and localStorage
<trigger click event>
check for expected state on element and localStorage

Reply:
I can split this test.